LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright {yyyy} {name of copyright owner}
+   Copyright 2016 Boundless, http://boundlessgeo.com
 
```

Comment:
thanks, looks great.

(Did all of this work happen in 2016? or is some of it 2015-2016...)